shell/platform/windows/flutter_windows_view.cc

```diff
@@ -25,17 +25,49 @@ constexpr std::chrono::milliseconds kWindowResizeTimeout{100};
 /// to be blocked until the frame with the right size has been rendered. It
 /// should be kept in-sync with how the engine deals with a new surface request
 /// as seen in `CreateOrUpdateSurface` in `GPUSurfaceGL`.
-bool SurfaceWillUpdate(size_t cur_width,
-                       size_t cur_height,
-                       size_t target_width,
-                       size_t target_height) {
+static bool SurfaceWillUpdate(size_t cur_width,
```

Comment:
style nit here and below: can't remember what precedent is in the windows embedder but consider using the C++-y convention of:
```cc
namespace {
  // ... stuff you want to be confined to this translation unit ...
}  // namespace
```

Reply:
Updated!